app/services/db-export/convert-to-csv.service.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+/**
+ * @module ConvertToCSVService
```

Comment:
This plus some other files are missing the description we normally add to `@module`